lms/templates/navigation.html

```diff
@@ -49,7 +49,6 @@ <h1 class="logo">
 
     % if course:
     <h2 class="course-header"><span class="provider">${course.display_org_with_default | h}:</span>
-      ${course.display_number_with_default | h}
```

Comment:
@antoviaque Yep, I saw that, but didn't have a chance to implement it that way so far.  Done now (but there is no upstream PR yet).